### DISCUSSION TITLE
Welcome to nwjs-ffmpeg-prebuilt Discussions!

### DISCUSSION THREAD

**Discussion by iteufel** [nwjs-ffmpeg-prebuilt/nwjs-ffmpeg-prebuilt#106] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by danrre** [comment]
Hi.
I'm a bit of a newbie to linux.
Please, could you help me to reverse/undo or remove the "npx nwjs-ffmpeg-prebuilt --arch x64" instruction, it didn't work for me.
Would it be enough to uninstall npm?

Thank you very much in advance.
Regards.
